ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,16 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+namespace DigitalRootFinder {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter the number:");
+         int n = int.Parse (Console.ReadLine ());
+         if (n != 0) {

```

Comment:
Digital root is defined only for positive numbers

Reply:
I will change with n>0 and update